templates/partners/desktop.html

```diff
@@ -1,154 +1,84 @@
 {% extends 'base_index.html' %}
 
 {% block content %}
-  <section class="p-strip--suru-desktop">
+  <section class="p-strip--suru-topped">
     <div class="row">
       <div class="col-6">
-        <h1>Desktop</h1>
-        <p>Partnering with Canonical is a great way to keep your products ahead of the competition. The Technical Partner Programme gives you access to information, tools and training that can help you get to market quicker and for less. And in retail and online, we work with our hardware partners on the promotion of Ubuntu to consumers and businesses.</p>
+        <h1>Desktop Programme</h1>
+        <p>Canonical collaborates with the world's leading vendors to optimise and certify Ubuntu on 100s of laptops, desktops and workstations for a secure and constantly updated platform.</p>
+        <p>Whether built with the developer in mind or to power AI ambitions, these out-of the-box platforms give practitioners everything they need to transform their enterprise IT.</p>
+        <a href="https://certification.ubuntu.com/desktop" class="p-button--neutral"><span class="p-link--external">Find a certified PC</span></a><a href="https://partners.ubuntu.com/contact-us" class="p-button--positive"><span class="p-link--external">Get in touch</span></a>
+      </div>
+      <div class="col-6 u-vertically-center">
+        <ul class="p-inline-images">
+          <li class="p-inline-images__item">
+            <img src="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg" alt="Dell" style="height: 75px;">
```

Comment:
any reason why you use inline style instead of html `width` and `height`?

Reply:
I was original doing a layout that needed more CSS but changed it and left these bits in... I should use HTML attributes.